installer/terraform/scmProvisioner.tf

```diff
@@ -6,22 +6,27 @@ resource "null_resource" "createProjectsInBB" {
+  }
 }
 
 // Copy the jazz-build-module to SLF in SCM
 resource "null_resource" "copyJazzBuildModule" {
-  depends_on = ["null_resource.postJenkinsConfiguration", "null_resource.createProjectsInBB"]
+  depends_on = ["null_resource.postJenkinsConfiguration", "null_resource.createProjectsInBB", "null_resource.updategitlabjenkinsconfig"]
 
   provisioner "local-exec" {
-    command = "${var.scmpush_cmd} ${var.scmgitlab == 1 ? join(" ", aws_lb.alb_ecs_gitlab.*.dns_name) : lookup(var.scmmap, "scm_elb") } ${lookup(var.scmmap, "scm_username")} ${lookup(var.scmmap, "scm_passwd")} ${var.cognito_pool_username} ${var.scmgitlab == 1 ? join(" ", data.external.gitlabcontainer.*.result.token) : lookup(var.scmmap, "scm_privatetoken") } ${var.scmgitlab == 1 ? join(" ", data.external.gitlabcontainer.*.result.scm_slfid) : lookup(var.scmmap, "scm_slfid") } ${lookup(var.scmmap, "scm_type")}  ${var.dockerizedJenkins == 1 ? join(" ", aws_lb.alb_ecs_jenkins.*.dns_name) : lookup(var.jenkinsservermap, "jenkins_elb") } ${lookup(var.jenkinsservermap, "jenkinsuser")} ${lookup(var.jenkinsservermap, "jenkinspasswd")} ${aws_api_gateway_rest_api.jazz-prod.id} ${var.region} builds"
+    command = "${var.scmpush_cmd} ${var.scmgitlab == 1 ? join(" ", aws_lb.alb_ecs_gitlab.*.dns_name) : lookup(var.scmmap, "scm_elb") } ${lookup(var.scmmap, "scm_username")} ${lookup(var.scmmap, "scm_passwd")} ${var.cognito_pool_username} ${var.scmgitlab == 1 ? join(" ", data.external.gitlabconfig.*.result.gitlab_token) : lookup(var.scmmap, "scm_privatetoken") } ${var.scmgitlab == 1 ? join(" ", data.external.gitlabconfig.*.result.gitlab_slfid) : lookup(var.scmmap, "scm_slfid") } ${lookup(var.scmmap, "scm_type")}  ${var.dockerizedJenkins == 1 ? join(" ", aws_lb.alb_ecs_jenkins.*.dns_name) : lookup(var.jenkinsservermap, "jenkins_elb") } ${lookup(var.jenkinsservermap, "jenkinsuser")} ${lookup(var.jenkinsservermap, "jenkinspasswd")} ${aws_api_gateway_rest_api.jazz-prod.id} ${var.region} builds"
   }
 }
 
 // Configure jazz-installer-vars.json and push it to SLF/jazz-build-module
 resource "null_resource" "configureJazzBuildModule" {
   depends_on = ["null_resource.copyJazzBuildModule"]
   provisioner "local-exec" {
-    command = "${var.pushInstallervars_cmd} ${lookup(var.scmmap, "scm_username")} ${urlencode(lookup(var.scmmap, "scm_passwd"))} ${var.scmgitlab == 1 ? join(" ", aws_lb.alb_ecs_gitlab.*.dns_name) : lookup(var.scmmap, "scm_elb") } ${lookup(var.scmmap, "scm_pathext")} ${var.cognito_pool_username}"
+    command = "${var.pushInstallervars_cmd} ${lookup(var.scmmap, "scm_username")} ${urlencode(lookup(var.scmmap, "scm_passwd"))} ${var.scmgitlab == 1 ? join(" ", aws_lb.alb_ecs_gitlab.*.dns_name) : lookup(var.scmmap, "scm_elb") } ${lookup(var.scmmap, "scm_pathext")} ${var.cognito_pool_username} ${lookup(var.scmmap, "scm_type")} ${var.scmgitlab == 1 ? join(" ", data.external.gitlabconfig.*.result.gitlab_token) : lookup(var.scmmap, "scm_privatetoken") }"
```

Comment:
@bleggett Got this error ^[[0m^[[0mnull_resource.configureJazzBuildModule (local-exec): Cloning into 'jazz-build-module'...
^[[0m^[[0mnull_resource.configureJazzBuildModule (local-exec): cp: cannot stat ‘/installer/terraform/provisioners/cookbooks/jenkins/files/default/jazz-installer-vars.json’: No such file or directory
^[[0m^[[0mnull_resource.configureJazzBuildModule (local-exec): fatal: pathspec 'jazz-installer-vars.json' did not match any files

Reply:
Does this fail the run? If it doesn't I might have missed it in the output of my scenario3 testing.

Reply:
Ah fixed. Somehow the script was using an env var to root the path, when it didn't really need to at all, can use a relative path like the other scripts do.